docs/minimum-iam-policies.md

```diff
@@ -0,0 +1,181 @@
+# Minimum IAM policies
+
+This document describes the minimum IAM policies needed to run the main use cases of eksctl. These are the ones used to
+run the integration tests.
+
+> **Note**: remember to replace `<account_id>` with your own.
+
+AmazonEC2FullAccess
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": "ec2:*",
+            "Effect": "Allow",
+            "Resource": "*"
```

Comment:
based on my experience, there might be some question about _exact_ permissions later :). FullAccess on some AWS services (e.g. ec2, lb, cloudwatch, ...) might not be allowed in some orgs such as my company. :)

Reply:
Good point. I don't know how to narrow it down right now so let's leave it like and when someone figures that out we can update it.